compiler/rustc_monomorphize/src/partitioning.rs

```diff
@@ -156,7 +156,6 @@ where
         placed
     };
 
-    // Merge until we have at most `max_cgu_count` codegen units.
```

Comment:
I suggest keeping the comment and replacing `max_cgu_count` with `cx.tcx.sess.codegen_units()`. Or if you don't like being that specific, changing it to "Merge until we don't exceed the max CGU count."